components/Dialogs/src/MDCAlertController.m

```diff
@@ -234,6 +234,15 @@ - (void)setTitleIconTintColor:(UIColor *)titleIconTintColor {
   }
 }
 
+- (void)setScrimColor:(UIColor *)scrimColor {
+  _scrimColor = scrimColor;
+  MDCDialogPresentationController *dialogPresentationController =
+      self.mdc_dialogPresentationController;
+  if (dialogPresentationController) {
```

Comment:
Nit: Please remove the `nil` check, [they shouldn't be used in Objective-C](http://google.github.io/styleguide/objcguide.html#nil-checks) unless it would cause a crash.